slsDetectorServers/ctbDetectorServer/CMakeLists.txt

```diff
@@ -18,12 +18,14 @@ add_executable(ctbDetectorServer_virtual
     ../slsDetectorServer/src/programFpgaBlackfin.c
     ../slsDetectorServer/src/sharedMemory.c
     ../slsDetectorServer/src/loadPattern.c
+    ../../slsSupportLib/opensslMd5/md5.c
 )
 
 include_directories(
     ../slsDetectorServer/include
     ../../slsSupportLib/include
     ../../slsDetectorSoftware/include/sls/
+    ../../slsSupportLib/opensslMd5
```

Comment:
maybe we can put the header in include/sls/md5/md5.h and the source in src/md5/md5.c? Then we wouldn't need the additional include path.